### ISSUE TITLE
Site is too small for viewport

### ISSUE THREAD

**Issue by farishkash** [Charn293/f1-3-c2p1-colmar-academy#1]
![colmar_academy](https://user-images.githubusercontent.com/5844161/38562791-3be43f30-3c90-11e8-851d-11bb6f4ef186.png)

The blue square is all the whitespace on my screen.

The goal of Colmar project is to be a full-screen project.

https://github.com/Charn293/f1-3-c2p1-colmar-academy/blob/077cb33fcaa47108d71b90ace2234d70a0ccc2be/Colmar/resources/css/style.css#L11

If you are using a laptop the screen may look fine without realizing that on other screens, such a typical Hight definition one, that this site is too small.

You may need to make additional CSS adjustments but after removing the max-width line, it looked fine.

![colmar_academy2](https://user-images.githubusercontent.com/5844161/38562785-383b437e-3c90-11e8-8517-25663b948bfa.png)

About 50 percent of the market is using a resolution of 1920 or wider.

https://www.w3schools.com/browsers/browsers_display.asp

**Comment by Charn293** [comment]
Yes, I did the project on a laptop. So I will watch this in future projects.
Thanks,
Chris.
